tests/Application/Actions/Donations/UpdateHandlesLockExceptionTest.php

```diff
@@ -136,4 +170,60 @@ public function request($method, $path, $params, $opts)
 
         return $stripeClient;
     }
+
+    public function getDonation(): Donation
```

Comment:
@bdsl Nice move externalising this into its own method to setup the test donation.